docs/reference/modules/asset.md

```diff
@@ -0,0 +1,214 @@
+
+#### Example
+
+Extending the previous example with an additional alias using a simple object:
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+// ...
+webpack: {
+ extensionOptions: {
+   addAlias: {
+       alias: {
+         Data: path.resolve(process.cwd(), './lib/data')
+       }
+     }
+   }
+ }
+},
+// …
+};
+```
+<template v-slot:caption>
+module/my-new-widget/index.js
+</template>
+</AposCodeBlock>
+
+### `bundles`
+The `bundles` option allows delivery of frontend assets to a specific page or if a widget is on a page page. The` bundles` subproperty can be present in any type of module, not just widget type and page type modules. However, configuring a bundle in a widget type or page type module is the only way to cause it to actually be loaded. If a bundle is present in another type of module, that module can contribute code to it, but it will only get loaded when also configured for at least one widget type or page type. This option takes an object containing either the JavaScript or stylesheet file names without extension as properties. These files should be located in the module's `ui/src` folder. Each property takes either an empty object or a sub-property of `templates`. The `templates` sub-property accepts an array of page template names where the content should be loaded. The `templates` feature is most often relevant for piece-page-type modules; if it is not used, it is assumed both `index` and `show` templates should load the bundle.
+
+Multiple modules can contribute to a single bundle by using the same bundle name, and webpack will merge these files in the final build. In addition, any shared dependencies between the extra bundles and the "main" bundle built from the code found in the `ui/src/index.js` files will be resolved so that only one instance is loaded.
+
+::: warning
+While extra bundles are a great feature, when used incorrectly they make sites slower, not faster.
+
+Always ask yourself this question: **will a typical site visitor eventually load this code?** If so, you should **leave it in the main bundle** (import it from `ui/src/index.js`). This way the frequently needed code is always loaded up front and reused by every page without an extra request to the server.
+
+Extra bundles should **only be used if the user probably won't need them on most visits.**
+:::
+
+
+## Related documentation
+
+- [Extending webpack configuration](/guide/webpack.md)
```

Comment:
Absolute links tend to fail in production with this doc builder.

`../../guide/webpack.md` I think.

Reply:
Changed. links are a little odd with this builder. I'm still having issues with getting the favicon displaying for the a2 docs.